constants/task.enum.js

```diff
@@ -6,7 +6,13 @@ export const PriorityEnum = {
 };
 
 export const StatusEnum = {
-  NEW: 1,
-  IN_PROGRESS: 2,
-  COMPLETED: 3
+  IN_PROGRESS: 1,
+  COMPLETED: 2
```

Comment:
What about new?

Reply:
Ok, I will use 'NEW' instead of 'IN_PROGRESS'.